athena-dynamodb/pom.xml

```diff
@@ -80,6 +80,19 @@
             <version>${log4j2Version}</version>
             <scope>runtime</scope>
         </dependency>
+        <dependency>
+            <groupId>com.amazonaws</groupId>
+            <artifactId>athena-federation-integ-test</artifactId>
+            <version>1.0</version>
+            <scope>test</scope>
```

Comment:
can you confirm that this module does _not_ end up in the final lambda jar? you can do this by unziping the final DDB jar and ensuring the classes from this module don't show up in it. Also confirm no CDK dependencies show up. This is important because jar size is a tough limit in lambda.

Reply:
confirmed.